src/sql/mod.rs

```diff
@@ -77,6 +79,11 @@ impl SqlIncorporator {
         }
     }
 
+    /// Make any future base nodes added be transactional.
+    pub fn make_transactional(&mut self) {
+        self.transactional = true;
+    }
```

Comment:
This presumably means that all future base nodes added will be transactional, whether `recipe::activate()` is called with the `transactional` flag set or not. Is there a reason why we need to make this a flag that can only be flipped once?